packages/mobile/src/redux/migrations.test.ts

```diff
@@ -385,4 +386,16 @@ describe('Redux persist migrations', () => {
     expect(migratedSchema2.walletConnect.v1).toBe(v18Schema.walletConnect.v1)
     expect(migratedSchema2.walletConnect.v2).toBeUndefined()
   })
+  it('works for v21 to v22', () => {
+    const migratedSchema = migrations[22](v21Schema)
+
+    expect(migratedSchema.i18n).toEqual({
+      language: v21Schema.app.language,
```

Comment:
nit: what if `v21Schema.app.language` was actually `undefined`? I know it's not the case here but it'd slightly better to use the actual string. in case it changes in the future we would spot it.